src/test/java/com/artipie/rpm/http/RpmSliceITCase.java

```diff
@@ -45,18 +52,23 @@
 import org.testcontainers.containers.GenericContainer;
 
 /**
- * Integration test for {@link RpmSlice}.
+ * Test for {@link RpmSlice}.
  * @since 0.10
  * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  */
 @SuppressWarnings("PMD.AvoidDuplicateLiterals")
-final class RpmSliceITCase {
+public final class RpmSliceITCase {
 
     /**
      * Vertx instance.
      */
     private static final Vertx VERTX = Vertx.vertx();
 
+    /**
+     * Settings files.
+     */
+    private final Path setting = Paths.get("src/test/resources/example.repo");
```

Comment:
@olenagerasimova it's not really reliable, since current working directory may differ from project root in case of maven builds with `-f` option or non-idea IDEs. It's better to access test resources via class-loader of current thread context and copy to `@TempDir`

Reply:
@g4s8 I've remade test to `fileSystemBind` on temp dir, so we don't need to access resources anymore. But I should fix this in the project, there are a lot of references to test resourses...